src/main/java/io/fabric8/elasticsearch/plugin/acl/SearchGuardRolesMapping.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.elasticsearch.plugin.acl;
+
+import static io.fabric8.elasticsearch.plugin.KibanaUserReindexFilter.getUsernameHash;
+import static io.fabric8.elasticsearch.plugin.acl.SearchGuardRoles.PROJECT_PREFIX;
+import static io.fabric8.elasticsearch.plugin.acl.SearchGuardRoles.ROLE_PREFIX;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.apache.commons.lang.builder.ToStringStyle;
+
+public class SearchGuardRolesMapping 
+	implements Iterable<SearchGuardRolesMapping.RolesMapping> {
+
+	private static final String USER_HEADER = "users";
+	private static final String ADMIN_ROLE = "sg_project_operations";
+	private List<RolesMapping> mappings;
+	
+	public static class RolesMapping {
+		
+		private String name;
+		
+		private List<String> users = new ArrayList<String>();
+		
+		public String getName() {
+			return name;
+		}
+		
+		public void setName(String name) {
+			this.name = name;
+		}
+		
+		public List<String> getUsers() {
+			return users;
+		}
+		
+		public void setUsers(List<String> users) {
+			this.users = users;
+		}
+		
+		public void addUser(String user) {
```

Comment:
scratch that, i was definitely thinking about another section of code that used 'addUser'... will remove (also didn't see with your comment that it was a blank method)